app/src/LightingManager.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "LightingManager.h"
+
+#include <lib/support/logging/CHIPLogging.h>
+#include <wiringPi.h>
+
+const int wiringPiPin = 7;
```

Comment:
For now, it is set to wiringPi pin 7 which is also the bcm pin 7: https://pinout.xyz/pinout/wiringpi
This needs to be configurable.